subseas_flex_fcst/config-bd.yaml

```diff
@@ -23,3 +23,21 @@ targets: null
 
 # Viz' set up
 zoom: 7
+
+# Bangladesh
+shapes_adm:
+       - name: Divisions
```

Comment:
This is indented by eight spaces. In other yaml files we mostly use four.

Are you using a different editor than usual? Maybe it can be configured.

Reply:
Only fixed this for now. I don't seem to be getting maps through devi on the hotel internet. Pushing to try my chance directly on my mac.